### ISSUE TITLE
Separate device instead of hat. 

### ISSUE THREAD

**Issue by mgrouch** [OpenMarine/MacArthur-HAT#11]
I think hat design is too much of vendor lock-in for users.

 It would be better to have an open source, open hardware multiplexer box. 
with few NMEA 0183, one NMEA 2000, Seatalk
and USB and Ethernet (tcp, udp) to connect to pi. Might be box with MCU (esp32 or something). 

Whole thing with waterproof box and proper ip67 connectors. 

Then might be some other simpler hat for pi with i2c, Victron connector, few temp sensors connectors, maiana (plug and button interface), uart plug for pypilot motor controller, RTC clock, powering pi from 12/24v.
Audio card with audio in/out (maybe). 
(Optional LoRa module?)

All NMEA coming from separate device via usb or Ethernet. 

Thanks

**Comment by mgrouch** [comment]
My experience tells me that targeted audience needs a ready to plug device. Those people just do not have time to deal with hats. 
they need a device and simple easy to follow setup instructions.

**Comment by masonleschyna** [comment]
I agree with this idea. Make a sensor hub with a single standard output. USB could work but you could also use NMEA OneNet. This HAT would theb serve as a gateway to the onboard sensors for devices connected via USB or LAN.

**Comment by mgrouch** [comment]
> I agree with this idea. Make a sensor hub with a single standard output. USB could work but you could also use NMEA OneNet. This HAT would theb serve as a gateway to the onboard sensors for devices connected via USB or LAN.

There some solutions (still half baked and non open source).

I can mention 

Victron CerboGX sensors support,

Pitufino NMEA gateway

ESP32 Sailor Hat

CX5003 nmea2000 module
